src/System.Data.SqlClient/src/System.Data.SqlClient.csproj

```diff
@@ -151,20 +154,20 @@
     <Compile Include="System\Data\SqlTypes\SqlXml.cs" />
     <Compile Include="$(CommonPath)\System\NotImplemented.cs" />
     <Compile Include="$(CommonPath)\System\Data\Locale\Locale.cs">
-        <Link>System\Data\Locale\Locale.cs</Link>
+      <Link>System\Data\Locale\Locale.cs</Link>
```

Comment:
The motivation behind having `Locale` in Common was because of InterOp calls into the Localization Dlls. 
Locale for Windows makes Interop Calls, however for Non-Windows it reads from a map. 
I wonder if any other libraries operate with LCIDs and don't expect them to call into System.Data.* Locale related functionality.